Zabbix_Meraki_Network_Discovery_Template.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<zabbix_export>
+    <version>5.2</version>
+    <date>2021-05-31T15:25:23Z</date>
+    <groups>
+        <group>
+            <name>Meraki_Autodiscover</name>
+        </group>
+        <group>
+            <name>Templates/Applications</name>
+        </group>
+        <group>
+            <name>Templates/Network devices</name>
+        </group>
+    </groups>
+    <templates>
+        <template>
+            <template>Meraki Network Discovery</template>
+            <name>Meraki Network Discovery</name>
+            <description>Template used to create Meraki Hosts automatically from Dashboard API export.</description>
+            <groups>
+                <group>
+                    <name>Templates/Applications</name>
+                </group>
+                <group>
+                    <name>Templates/Network devices</name>
+                </group>
+            </groups>
+            <discovery_rules>
+                <discovery_rule>
+                    <name>Meraki Host Creation</name>
+                    <type>EXTERNAL</type>
+                    <key>meraki.sh[network.discovery]</key>
+                    <delay>12h</delay>
+                    <lifetime>7d</lifetime>
+                    <host_prototypes>
+                        <host_prototype>
+                            <host>Meraki {#NETWORK_NAME}</host>
+                            <name>Meraki {#NETWORK_NAME}</name>
+                            <group_links>
+                                <group_link>
+                                    <group>
+                                        <name>Meraki_Autodiscover</name>
+                                    </group>
+                                </group_link>
+                            </group_links>
+                            <group_prototypes>
+                                <group_prototype>
+                                    <name>Meraki/{#ORG}</name>
+                                </group_prototype>
+                            </group_prototypes>
+                            <templates>
+                                <template>
+                                    <name>AL Meraki Device Discovery</name>
```

Comment:
Incorrect template name for use with the included device discovery template.
This will fail on import.